bindings/java/src/main/com/apple/foundationdb/ReadTransaction.java

```diff
@@ -519,6 +527,7 @@ AsyncIterable<MappedKeyValue> getMappedRange(KeySelector begin, KeySelector end,
 	 *
 	 * @param begin beginning of the range (inclusive)
 	 * @param end end of the range (exclusive)
+	 * @param rowLimit -- undocumented
```

Comment:
this should mean "the limit on the number of returned rows".

Reply:
pushed an update for this.